libweasyl/libweasyl/conftest.py

```diff
@@ -31,11 +31,23 @@ def setup(request):
 @pytest.fixture(autouse=True)
 def staticdir(tmpdir):
     tmpdir = tmpdir.join('libweasyl-staticdir')
+    staff_dict = {
+        # [Fiz, Ikani]
+        'directors': [1014, 2061],
+        # [Weykent]
+        'technical_staff': [5756],
+        # [Hendikins, Kihari]
+        'admins': [23613, 3],
+        # [pinardilla]
+        'mods': [40212],
+        # [8BitFur, Charmander, Foximile, Kailys, Kauko]
+        'developers': [38623, 34165, 15224, 2475, 8627],
+    }
```

Comment:
I'd suggest making `staff_dict` an empty dict by default, since it's unlikely most of these user ids will be present during any given test.